src/ArraySetops.chpl

```diff
@@ -108,8 +102,7 @@ module ArraySetops
         var truth = makeDistArray(a.size, bool);
 
         // based on size of array, determine which method to use 
-        if (b.size <= sBound) then truth = in1dGlobalAr2Bcast(a, b);
-        else if (b.size <= mBound) then truth = in1dAr2PerLocAssoc(a, b);
+        if (b.size <= mBound) then truth = in1dAr2PerLocAssoc(a, b);
```

Comment:
Since you're optimizing is it worth it to make this a ternary operation for assignment (including the inversion of truth = !truth)?
i.e.
```chapel
var truth = if(b.size <= mBound) then !in1dAr2PerLocAssoc(a, b) else !in1dSort(a,b);
```

Although maybe separating the truth inversion would be more readable
```chapel
var truth = if(b.size <= mBound) then in1dAr2PerLocAssoc(a, b) else in1dSort(a,b);
truth = !truth;
...
```

Reply:
Yeah, I like this suggestion and I think it'd be even faster to pass `invert` into the core in1d routines so we can negate while iterating instead of having to take a second pass over the array.

I would like to put this in another PR though. I want to focus on communication/scaling improvements for the medium size in this particular PR and followup with additional optimizations later.

I added this to the next steps captured at the end of https://github.com/Bears-R-Us/arkouda/issues/970#issuecomment-1018649457